astropy/units/photometric.py

```diff
@@ -15,18 +15,19 @@
 
 import numpy as np
 
-from astropy.constants import si as _si
+from astropy.constants import L_bol0
```

Comment:
I think it matters for decomposition what units what units are in the namespace.

Reply:
Not sure I understand that. For `__all__` and `__doc__` it does matter that we do not import any unit anywhere, but here we have `Constant`, so it is OK.